pkg/api/vlabs/validate.go

```diff
@@ -802,16 +808,6 @@ func validateVMSS(o *OrchestratorProfile, isUpdate bool, storageProfile string)
 		if sv.LT(minVersion) {
 			return errors.Errorf("VirtualMachineScaleSets are only available in Kubernetes version %s or greater. Please set \"orchestratorVersion\" to %s or above", minVersion.String(), minVersion.String())
 		}
-		// validation for instanceMetadata using VMSS with Kubernetes
```

Comment:
This looks unrelated to the rest of the PR. Is this just tech debt removal or am I missing something?

Reply:
Yeah, possibly rebase bogon, addressing.